wagtailio/static/sass/components/_sign-up-form.scss

```diff
@@ -0,0 +1,132 @@
+.sign-up-form {
+    $root: &;
+    @include sf-spacing();
+
+    &__heading {
+        // normalise the h2 element
+        margin: 0;
+    }
+
+    &__sub-heading {
+        padding: 10px 0 0 0;
+
+    }
+
+    &__label {
+        display: none;
+    }
+
+    &__icon {
+        display: none;
+
+        @include media-query(medium) {
+            display: inherit;
+        }
+
+        > svg {
+            inset: -5% 3% auto auto;
+            width: 170px;
+            height: 253px;
+
+            @include media-query(large) {
+                inset: 10% 86px auto auto;
```

Comment:
Suggestion: add `position: relative` to `.sign-up-form` so the icon is positioned relative to that (it looks a little low for me at the moment)

Reply:
Sure thing.